packages/rum/test/e2e/manual-timing/app.e2e-spec.js

```diff
@@ -28,9 +28,6 @@ const { verifyNoBrowserErrors } = require('../../../../../dev-utils/webdriver')
 describe('manual-timing', function() {
   it('should run manual timing', async function() {
     browser.url('/test/e2e/manual-timing/index.html')
-    browser.setTimeout({
```

Comment:
The reason why I removed it because the latest appium version in saucelabs seems to have bug and there is no workaround other than using the default script timeout of 30 seconds. https://w3c.github.io/webdriver/#timeouts

Reply:
Lol. IE uses 0 seconds and fails.. Trying another way.